.gitignore

```diff
@@ -6,4 +6,19 @@ __pycache__/
 *.swp
 *~
 
+# cookie cutter
+{{cookiecutter.project_name}}/requirements-dev.txt
 {{cookiecutter.project_name}}/requirements.txt
+{{cookiecutter.project_name}}/static/images/
+
+# Editor files
```

Comment:
I feel like someone's local editor files belong in their local global git ignore file. Not that I can recall where that is right now, or if we do that for other projects. Don't feel super strongly on this one.

Reply:
Yep that's fair. It's also already in the .gitignore in the CC, so will review what's in there, there is probably a 'standard' python one that could be used, will have a look.

Reply:
Also this isn't a complete list. Our other .gitignores are quite cluttered with these sort of things.

That said, perhaps better exclude them here than accidentally get one merge? Also don't feel strongly.

Reply:
How about going with https://github.com/github/gitignore/blob/master/Python.gitignore (for both the template and the cut project) then it's fairly standard. 

Note that that doesn't include environments, so that's likely a better idea to have locally as you said.

Reply:
Used http://gitignore.io/api/django for now, probably a good place to then refine from, there is some stuff we likely don't need in there.